platform/android/export/export_plugin.cpp

```diff
@@ -1612,11 +1612,11 @@ Vector<String> EditorExportPlatformAndroid::get_enabled_abis(const Ref<EditorExp
 
 void EditorExportPlatformAndroid::get_preset_features(const Ref<EditorExportPreset> &p_preset, List<String> *r_features) {
 	String driver = ProjectSettings::get_singleton()->get("rendering/driver/driver_name");
-	if (driver == "OpenGL3") {
+	if (driver == "opengl3") {
```

Comment:
Shouldn't we use `etc2` for GLES3? This was likely done this way when it was GLES2 but this changed since then.

Reply:
Can be looked into in another PR though, all export preset features need to be rechecked.

Reply:
Indeed, this may be better left to a future PR.

I think we'll be able to remove ETC1 support entirely and focus on ETC2 as a baseline, which will simplify things a lot. That is, if no blocker bugs are found in mobile/web support of ETC2-compressed textures…